cpu/lpc1768/include/periph_cpu.h

```diff
@@ -65,6 +65,11 @@ typedef enum {
 /** @} */
 #endif /* ndef DOXYGEN */
 
+/**
+ * @brief   Number of available timer channels
+ */
+#define TIMER_CHANNEL_NUMOF     (4U)
```

Comment:
Apparently the boards define `TIMER_0_CHANNELS`, which has a value `1`, `2`, `3`, or `4` depending on the board.

Unless I missed something, this macro should be renamed instead.